### PR TITLE
[FC-112438] Choice name should not exceed 255 characters error message not shown in UI

### PR DESCRIPTION
**Pull request by Srikannan-E**
## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)

### CODE REVIEW

**Review by srivatsan7 — APPROVED** [review]
LGTM

**Review by prasanna-vijayan — APPROVED** [review]
Looks good to me. Can we have a detailed way how you did testing for this in UI?

File: packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx [hunk 1]
```diff
@@ -196,6 +230,8 @@ export class FbFieldDropdownItem {
       return null;
     }
 
+    const objMaxLimits = getMaximumLimitsConfig(this.productName);
```
Comment (rihansiddhi) [inline packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx]: @Srikannan-E Can you please use getMaxLimitProperty and get the count?

**Review by rihansiddhi — CHANGES_REQUESTED** [review]

File: packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx (hunk 1 above)
Comment (Srikannan-E) [reply packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx]: I done the changes bro

File: packages/crayons-extended/custom-objects/src/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
   </head>
 
   <body>
-    <!-- <div
+    <div
```
Comment (rihansiddhi) [inline packages/crayons-extended/custom-objects/src/index.html]: @Srikannan-E Why is this changed?

File: packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx [hunk 3]
```diff
@@ -225,6 +273,7 @@ export class FbFieldDropdownItem {
       strDragClassName += ` ${strBaseDragClassName}--unsortable`;
     }
 
+    const numNameMaxChars = objMaxLimits?.['maxChoiceChars']?.count || 255;
```
Comment (rihansiddhi) [inline packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx]: @Srikannan-E You can directly access count here instead of `objMaxLimits?.['maxChoiceChars']?.count`. Can you please update and test this one?

**Review by rihansiddhi — CHANGES_REQUESTED** [review]

File: packages/crayons-extended/custom-objects/src/components.d.ts [hunk 4]
```diff
@@ -125,6 +125,10 @@ export namespace Components {
           * Item selected based on id
          */
         "itemSelected": boolean;
+        /**
+          * The db type used to determine the json to be used for CUSTOM_OBJECTS or CONVERSATION_PROPERTIES
```
Comment (srivatsan7) [inline packages/crayons-extended/custom-objects/src/components.d.ts]: Revert these changes @Srikannan-E

**Review by srivatsan7 — CHANGES_REQUESTED** [review]

File: packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx (hunk 3 above)
Comment (Srikannan-E) [reply packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx]: All changes done bro

File: packages/crayons-extended/custom-objects/src/components.d.ts (hunk 4 above)
Comment (Srikannan-E) [reply packages/crayons-extended/custom-objects/src/components.d.ts]: reverted bro

**Review by rihansiddhi — APPROVED** [review]

**Review by srivatsan7 — APPROVED** [review]
